client/state/sharing/publicize/schema.js

```diff
@@ -0,0 +1,55 @@
+export const fetchingConnectionsSchema = {
```

Comment:
I'm more inclined toward the latter too. I've opened up https://github.com/Automattic/wp-calypso/pull/3372 to fix sites naming.